### ISSUE TITLE
Updated OpenSearch Lucene snapshot location

### ISSUE THREAD

**Issue by peternied** [opensearch-project/opensearch-hadoop#385]
### Description
This repository has dependencies on Lucene snapshots from OpenSearch that are hosted on cloudfront.  When these snapshots are updated the they can be unavailable between file deletion and re-upload causing build failures.  By switching to a proper hosting environment this will better ensure the availability of these dependencies.

### Recommendation
- Find `https://d1nvenhzbhpy0q.cloudfront.net/snapshots/lucene/`
- Replace `https://artifacts.opensearch.org/snapshots/lucene/`

### Additional Context
- Original Issue https://github.com/opensearch-project/opensearch-build/issues/3874
- Example pull request https://github.com/opensearch-project/OpenSearch/pull/11728

**Comment by dblock** [comment]
Makes sense. Want to fix this @peternied? Catch All Triage - [1](https://github.com/dblock) [2](https://github.com/krisfreedain) [3](https://github.com/gkamat) [4](https://github.com/andrross) [5](https://github.com/getsaurabh02)
